### PR TITLE
amp-bind: Remove web-worker fallback code

### LINKED ISSUE #8955: Clean up "web-worker" experiment
- Remove "web-worker" experiment checks and fallback from code.

Follow-up: Once above is in prod for awhile, remove "web-worker" experiment itself.

### PR DESCRIPTION
**Pull request by choumx**
Fixes #8955.

### CODE REVIEW

File: extensions/amp-bind/0.1/test/test-bind-evaluator.js [hunk 1]
```diff
@@ -69,6 +77,22 @@ describe('BindEvaluator', () => {
...
+    }, {
+      tagName: 'DIV',
+      property: 'text',
+      expressionString: '1+1',
+    }]);
+    const stub = sandbox.stub(BindExpression.prototype, 'evaluate');
+    stub.returns('stubbed');
+    evaluator.evaluateBindings({});
+    expect(stub.calledOnce).to.be.true;
+  });
```
Comment (choumx) [inline extensions/amp-bind/0.1/test/test-bind-evaluator.js]: This was moved from `test-bind-impl.js`.

File: extensions/amp-bind/0.1/test/test-bind-impl.js [hunk 2]
```diff
@@ -136,32 +119,32 @@ describes.realWin('Bind', {
   it('should throw error if experiment is not enabled', () => {
     toggleExperiment(env.win, 'amp-bind', false);
     // Experiment check is bypassed on test mode -- make sure it isn't.
-    window.AMP_MODE = {test: false};
+    env.sandbox.stub(getMode(), 'test', false);
```
Comment (kmh287) [inline extensions/amp-bind/0.1/test/test-bind-impl.js]: Is this the preferred way to stub the mode?

Comment (choumx) [inline extensions/amp-bind/0.1/test/test-bind-impl.js]: Setting `window.AMP_MODE` is more common but stubbing avoids overwriting other mode properties (e.g. line 180). 

IMO stubbing is generally better practice than setting global vars, but maybe there's something I'm missing. /cc @jridgewell

Comment (jridgewell) [inline extensions/amp-bind/0.1/test/test-bind-impl.js]: Use `AMP_MODE`, we exported it specifically for this. You can use `Object.assign` if you're worried about conflicting properties.

**Pull request merged**